docs/guide/gpsd-socket.md

```diff
@@ -0,0 +1,28 @@
+# GPSd time source
+Instead of using another NTP server as a time source, it is also possible to use time data from [GPSd](https://gpsd.gitlab.io/gpsd/) as a time source for ntpd-rs.
+GPSd is able to interpret GPS signals from a GPS receiver, from which it can derive the current time.
+
+GPSd can be added as a time source for ntpd-rs by adding the following to the configuration:
+```toml
+[[source]]
+mode = "sock"
+path = "/run/chrony.ttyAMA0.sock"
+measurement_noise_estimate = 0.001
+```
+The `path` points to the location of the socket that GPSd writes timing data to. This socket was originally meant for chrony, but ntpd-rs also supports this same socket format. Here, `ttyAMA0` is the GPS receiver device used by GPSd.
+
+The `measurement_noise_estimate` gives a static estimate of how noisy GPSd's timing data is. Normally with NTP time sources, we would use the network delay as an independent estimate of how noisy the data is. When using GPSd as a time source, we do not have a good estimate of the noise, so we use a static noise estimate instead.
```

Comment:
Is the estimate of 0.001 based on something? Is there a way to know what a good estimate would be? If yes, it might be nice to name it here. If no, this sentence is fine as-is